### PR TITLE
fix: update usage info

### PR DESCRIPTION
**Pull request by jamesgeorge007**
**What kind of change does this PR introduce?**
patch

**Did you add tests for your changes?**
Update test snapshots

**If relevant, did you update the documentation?**
Yes

**Summary**
#2576

**Does this PR introduce a breaking change?**
Nope

**Other information**
N/A

### CODE REVIEW

**Review by alexander-akait — APPROVED** [review]

File: OPTIONS.md [hunk 1]
```diff
@@ -859,9 +859,9 @@ Commands:
   help|h [command] [option]                                                          Display help for commands and options.
   info|i [options]                                                                   Outputs information about your system.
   init|c [generation-path] [options]                                                 Initialize a new webpack project.
-  loader|l [output-path]                                                             Scaffold a loader.
+  loader|l [output-path] [options]                                                             Scaffold a loader.
   migrate|m <config-path> [new-config-path]                                          Migrate a configuration to a new version.
-  plugin|p [output-path]                                                             Scaffold a plugin.
+  plugin|p [output-path] [options]                                                            Scaffold a plugin.
   serve|s [entries...] [options]                                                     Run the webpack dev server.
```
Comment (snitin315) [inline OPTIONS.md:865]: Can we fix the alignment as well?

Comment (alexander-akait) [reply OPTIONS.md:865]: We need to set TTY and columns https://github.com/tj/commander.js/blob/master/index.js#L577

Comment (snitin315) [reply OPTIONS.md:865]: Let's do it 👍🏻

**Review by snitin315 — APPROVED** [review]

File: packages/webpack-cli/README.md [hunk 2]
```diff
@@ -68,9 +68,9 @@ npx webpack-cli --help verbose
   help|h [command] [option]                  Display help for commands and options.
   info|i [options]                           Outputs information about your system.
   init|c [generation-path] [options]         Initialize a new webpack project.
-  loader|l [output-path]                     Scaffold a loader.
+  loader|l [output-path] [options]           Scaffold a loader.
   migrate|m <config-path> [new-config-path]  Migrate a configuration to a new version.
-  plugin|p [output-path]                     Scaffold a plugin.
+  plugin|p [output-path] [options]           Scaffold a plugin.
```
Comment (alexander-akait) [inline packages/webpack-cli/README.md:73]: Was it fixed manually? Because if we have problems with indent, we should fix it

Comment (jamesgeorge007) [reply packages/webpack-cli/README.md:73]: > Was it fixed manually?

Yup

Comment (alexander-akait) [reply packages/webpack-cli/README.md:73]: We need fix it, I will look at this

Comment (jamesgeorge007) [reply packages/webpack-cli/README.md:73]: I added the entry manually at first which broke the indentation. It got fixed once I grabbed the help output. I think we are good here.

Comment (alexander-akait) [reply packages/webpack-cli/README.md:73]: Yep you are right

**Review by alexander-akait — APPROVED** [review]

**Pull request merged**